ccmlib/node.py

```diff
@@ -1083,6 +1083,24 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
             stress_options = stress_options[:]
 
         stress = common.get_stress_bin(self.get_install_dir())
+
+        # handle cassandra-stress >= 3.11 parameters
+        has_limit = any(['limit=' in o for o in stress_options])
+        has_throttle = any(['throttle=' in o for o in stress_options])
+        if has_limit or has_throttle:
+            args = [stress, 'version']
+            p = subprocess.Popen(args, cwd=common.parse_path(stress),
+                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                                 **kwargs)
+            stdout, stderr = p.communicate()
+
+            if p.returncode == 1 and has_throttle:
```

Comment:
Why that? Does the c-s > 3.11 have no "version" argument?

Reply:
exactly opposite, c-s> 3.11 has *version* argument and  c-s<3.11 doesn't have *version* argument